common/inc/sgx_tseal.h

```diff
@@ -47,18 +47,18 @@
 
 typedef struct _aes_gcm_data_t
 {
-    uint32_t  payload_size;                   /*  0: Size of the payload which includes both the encrypted data and the optional additional MAC text */
-    uint8_t   reserved[12];                   /*  4: Reserved bits */
+    uint32_t  payload_size;                   /*  0: Size of the payload which includes the encrypted data: payload[] */
```

Comment:
The payload[] includes encrypted data and the optional additional authenticated data if users seal data with a plaintext (for example, `sgx_seal_data()` is called with `p_additional_MACtext` not NULL). So I suppose the previous annotation is correct.